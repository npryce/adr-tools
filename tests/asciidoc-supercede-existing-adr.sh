echo "adoc" > .adr-file-type
adr new First Record
adr new -s 1 Second Record
head -10 doc/adr/0001-first-record.adoc
head -12 doc/adr/0002-second-record.adoc

