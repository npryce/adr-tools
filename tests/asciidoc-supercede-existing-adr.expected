echo "adoc" > .adr-file-type
adr new First Record
doc/adr/0001-first-record.adoc
adr new -s 1 Second Record
doc/adr/0002-second-record.adoc
head -10 doc/adr/0001-first-record.adoc
= 1. First Record

Date: 1992-01-12

== Status

Superceded by link:0002-second-record.adoc[2. Second Record]

== Context

head -12 doc/adr/0002-second-record.adoc
= 2. Second Record

Date: 1992-01-12

== Status

Accepted

Supercedes link:0001-first-record.adoc[1. First Record]

== Context


